README.md

```diff
@@ -2,3 +2,27 @@
 
 [![Travis Build Status](https://travis-ci.org/libmpack/libmpack-lua.svg?branch=master)](https://travis-ci.org/libmpack/libmpack-lua)
 
+## Building
+
+```bash
+LUA_TARGET=$(PLATFORM) make
+```
+
+Where `PLATFORM` is a supported platform for lua 5.1, i.e. one of:
+  * aix
+  * ansi
+  * bsd
+  * freebsd
+  * generic
+  * linux
+  * macosx
+  * mingw
+  * posix
+  * solaris
```

Comment:
This is a lot of space dedicated to a pretty niche use-case. Is there maybe a Lua source you can just link to instead of listing everything here? Or perhaps just list this in parens instead of a bullet list.

Reply:
I'll update it to put them in parenthesis (likely tomorrow morning).

I tried finding a source to link to but I can't find one for lua 5.1 and the relevant page for [lua 5.4](https://www.lua.org/manual/5.4/readme.html) lists targets that aren't available in 5.1 (e.g. `guess`)

Reply:
I could also drop the full list, just give one example, and tell users to `cd .deps/src/5.1/lua && make` for a full list of platforms